ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -383,6 +390,7 @@ PrjFS_Result PrjFS_WritePlaceholderFile(
     
     if (nullptr == relativePath)
     {
+        LogWarning("PrjFS_WritePlaceholderFile: paramater relativePath should not be [NULL]");
```

Comment:
I've now looked at the context of this function a bit more and I'm thinking that at least for this particular case of `relativePath` being `nullptr`, this seems like a programmer error (misuse of the API) which should just bring down the process (via `assert()`) rather than a true runtime error that should be logged.

Reply:
We generally try to avoid exiting unless we are in an unrecoverable bad state.

Up the [call stack](https://github.com/microsoft/VFSForGit/blob/aff5bb1f304afe326d218ccefc4b27c4e6795ae9/GVFS/GVFS.Platform.Mac/MacFileSystemVirtualizer.cs#L635)

We log this as an error which will go to telemetry.  We monitor that telemetry and can act accordingly.  Since at a higher level this is logged as an Error I think it's good to keep this as a Warning here to avoid double entries in telemetry.